src/components/MessageInput/hooks/messageInput.js

```diff
@@ -665,8 +665,11 @@ export default function useMessageInputState(props) {
     [uploadNewFiles, insertText],
   );
 
+  const isUploadEnabled = channel?.getConfig?.()?.uploads !== false;
```

Comment:
So we want it to be considered enabled if for some reason we can't get the config explicitly, is that it?

Reply:
correct, in case the config is undefined we assume the features are supported